### DISCUSSION TITLE
Right click - close all other tabs

### DISCUSSION THREAD

**Discussion by giwul70** [sumatrapdfreader/sumatrapdf#3456] [Ideas]
First of all - I have been using SumatraPDF only for a short while now, so bear with me when the below has been asked before. I may be overlooking earlier threads. 
Secondly: SumatraPDF is a great tool. As a matter of fact I have a license on Acrobat, but for everyday use I swapped it for SumatraPDF (e.g. files are associated with SumatraPDF). I use Acrobat for some specific tasks only.

Please allow me to suggest an option to r-click an open tab and add an option to "Close all other tabs".
This in case of multiple tabs of course.
Right now I need to close all tabs one by one. Maybe there are faster ways, I don't know.

Thank you.
(SumatraPDF v3.4.6 x64)

**Comment by tactilis** [comment]
Already implemented in the pre-release version:

![image](https://user-images.githubusercontent.com/4060092/233603484-f08ccd23-06d8-4d0a-8481-fff98e222c1a.png)

**Reply by giwul70** [reply]
Ah! THANK you.
That's wonderful news.

Keep up the good work!

Thanks again.
